test/assets/site_create.xml

```diff
@@ -1,4 +1,4 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <tsResponse xmlns="http://tableau.com/api" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://tableau.com/api http://tableau.com/api/ts-api-2.3.xsd">
-    <site id="0626857c-1def-4503-a7d8-7907c3ff9d9f" name="Tableau" contentUrl="tableau" adminMode="ContentAndUsers" userQuota="15" disableSubscriptions="true" state="Active" revisionHistoryEnabled="false" subscribeOthersEnabled="true" />
+    <site id="0626857c-1def-4503-a7d8-7907c3ff9d9f" name="Tableau" contentUrl="tableau" adminMode="ContentAndUsers" storageQuota="15" disableSubscriptions="true" state="Active" revisionHistoryEnabled="false" subscribeOthersEnabled="true" allowSubscriptionAttachments="true" guestAccessEnabled="false" cacheWarmupEnabled="true" commentingEnabled="true" editingFlowsEnabled="false" schedulingFlowsEnabled="false" extractEncryptionMode="enabled" catalogingEnabled="true" derivedPermissionsEnabled="false" requestAccessEnabled ="false" runNowEnabled="true" numUsers="0" storage="0" userQuota="15" tierCreatorCapacity="2" tierExplorerCapacity="1" tierViewerCapacity="1" isDataAlertsEnabled="true" askDataMode="EnabledByDefault" useDefaultTimeZone="false" timeZone="America/Los_Angeles"/>
```

Comment:
Could you also add verification of the field values in test_site.py?
There's a lot of new fields, so you can spread them across test cases. The importance is to have test coverage to ensure each field is parsed correctly.